templates/data/mongodb/support.html

```diff
@@ -0,0 +1,429 @@
+      <div class="col-9 col-medium-6">
+        <div class="row p-section--shallow">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">MongoDB architecture consulting</h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Work with a Canonical platform architect to design the open source data solution that works best for you. Design services covering requirements analysis, environment planning, initial dimensioning and solution design. Fully documented and aligned with your needs.
+            </p>
+          </div>
+        </div>
+        <hr class="is-muted" />
+        <div class="row p-section--shallow">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">MongoDB deployment service</h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Canonical engineers deploy Charmed MongoDB to the cloud or data centre of your choice. Kubernetes and air-gapped environments are fully supported.
+            </p>
+            <hr class="is-muted" />
+            <p>
+              <a href="https://assets.ubuntu.com/v1/aeb556fb-MongoDB%20-%20Datasheet-revised.pdf">Download
+              the datasheet&nbsp;&rsaquo;<span class="u-off-screen">Download the datasheet</span></a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row">
+      <hr />
+      <div class="col-6 col-medium-3">
+        <h2>Charmed MongoDB training</h2>
+      </div>
+      <div class="col-6 col-medium-3">
+        <p>
+          Get hands-on training with MongoDB and Charmed MongoDB, Canonical’s solution for MongoDB, in a three day workshop delivered remotely or on-site.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a class="p-button"
+             href="/contact-us"
+             aria-controls="data-mongodb-modal">Get in touch<span class="u-off-screen">Get in touch</span></a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <p class="p-heading--2">
+        <a href="/contact-us" aria-controls="data-mongodb-modal">Contact us to get support for MongoDB&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col p-section">
+        <h2>Come for MongoDB support, access all Canonical solutions</h2>
+      </div>
+      <div class="col p-section">
+        <p>
+          With Ubuntu Pro + Support for MongoDB, you get an all-access pass to everything Canonical has to offer, including best-in-class open source solutions for enterprise data management and AI. Ubuntu Pro subscriptions are competitively priced per server, not per application.
+        </p>
+      </div>
+      <div class="p-logo-section--dense">
```

Comment:
These logos look extremely small, did design confirm the sizing here?
![image](https://github.com/user-attachments/assets/c8c1a3f6-425a-4f49-8b21-8fa32df2f792)

Reply:
Lyubo suggested me to do this way [here](https://canonical.com/data/opensearch/support). This has to be corrected in Vanilla, next version should fix this.